my_list.rb

```diff
@@ -0,0 +1,26 @@
+require_relative 'my_enumerable'
+class Mylist
+  include MyEnumerable
+  def initialize(*list)
+    @list = list
+  end
+
+  def each(&block)
+    @list.each(&block)
+  end
```

Comment:
- [ ] For this module, we actually expect you to differently create your own `each` method from scratch instead of using the specified inbuilt array method. It should be similar to the example below:

```ruby
class MyList

[...]

def each
    return to_enum(:each) unless block_given?

    counter = 0
    while counter < @list.length
      yield(@list[counter])
      counter += 1
    end
    @list
end

[...]

end
```

Reply:
@rloterh !
Thank you for reviewing my code. I will change it.